changelog.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+## [2.0] - 2019-12-04
+### Added
+1. `UserInterface` for all user entities to implement.
+### Changed
+1. `AuthInterface` user method updated to return `UserInterface` instead of `EntityInterface`  
```

Comment:
This one needs to be tagged as a BC BREAK

Reply:
done.